engine/eiface.h

```diff
@@ -359,7 +359,7 @@ typedef enum _fieldtypes
 #define offsetof(s,m)	(size_t)&(((s *)0)->m)
 #endif
 
-#define _FIELD(type,name,fieldtype,count,flags)		{ fieldtype, #name, offsetof(type, name), count, flags }
+#define _FIELD(type,name,fieldtype,count,flags)		{ fieldtype, #name, (int) offsetof(type, name), count, flags }
```

Comment:
It's not a good idea.
And it was fixed in master branch.
https://github.com/FWGS/hlsdk-xash3d/commit/cf6647e07c452f6522aa8b5a9619a1ac1246f9af

Reply:
OK